fastlane/Fastfile

```diff
@@ -542,7 +618,7 @@ platform :ios do
     archive_zip_path = File.join(File.dirname(Dir.pwd), 'WooCommerce.xarchive.zip')
     zip(path: lane_context[SharedValues::XCODEBUILD_ARCHIVE], output_path: archive_zip_path)
 
-    version = options[:beta_release] ? ios_get_build_version : app_version
+    version = options[:beta_release] ? current_build_code : current_release_version
     create_release(
       repository: GITHUB_REPO,
       version:,
```

Comment:
Missing a value for this parameter here 🤔

Reply:
That's the Ruby 3 convention of not needing to include the argument if the name of the argument is the same as the name of the parameter.